controllers/endpoints.controller.js

```diff
@@ -0,0 +1,12 @@
+const { fetchAvailableEndpoints } = require("../models/endpoints.model");
+
+
+module.exports.availableEndpoints = (req, res, next) => {
+  fetchAvailableEndpoints()
+    .then((endpoints) => {
+      res.status(200).send({ endpoints });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
- Good work
- Sending the response back on a key of `endpoints` is a good decision and well named.